src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-split-panel-trigger-tooltip.test.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import createWrapper from '../../../../lib/components/test-utils/selectors/index';
+import { drawerIds as drawerIdObj } from '../../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT } from '../constants';
+import { AppLayoutDrawersPage, setupTest } from '../utils';
+
+import toolbarStyles from '../../../../lib/components/app-layout/visual-refresh-toolbar/toolbar/styles.selectors.js';
+
+const wrapper = createWrapper().findAppLayout();
+
+const drawerIds = Object.values(drawerIdObj);
+
+describe('refresh-toolbar', () => {
+  const theme = 'refresh-toolbar';
+  const mobileDrawerTriggerIds = drawerIds.slice(
+    0,
+    VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT + (theme === 'refresh-toolbar' ? 1 : 0)
+  );
+  const appliedThemeStyles = toolbarStyles; //use visualRefreshStyles when theme === 'refresh'
+
+  describe.each(['desktop', 'mobile'] as const)('%s', size => {
+    const drawersTriggerContainerClassKey = `drawers-${size === 'desktop' ? 'desktop' : 'mobile'}-triggers-container`;
+    const drawerIdsToTest = size === 'mobile' ? mobileDrawerTriggerIds : drawerIds; //toolbarDrawerIds;
+    const firstDrawerTriggerSelector = wrapper.findDrawerTriggerById(drawerIdsToTest[0]).toSelector();
+    const splitPanelTriggerSelector = wrapper.findSplitPanelOpenButton().toSelector();
+    const tooltipSelector = wrapper.findDrawerTriggerTooltip().toSelector();
+    const expectedTooltipText = 'Open panel';
+
+    describe.each(['bottom', 'side'] as const)('splitPanelPosition=%s', splitPanelPosition => {
```

Comment:
What is the point of testing this behavior for both `bottom` and `side` view? Are they different?

Reply:
yes. I observed different behaviors happening and want to confirm the behaviors are consistent